language/tools/move-package/tests/test_sources/compilation/basic_no_deps/Move.exp

```diff
@@ -0,0 +1,10 @@
+CompiledPackageInfo {
+    package_name: Identifier(
+        "test",
+    ),
+    address_alias_instantiation: {},
+    module_resolution_metadata: {},
+    source_digest: Some(
+        "ELIDED_FOR_TEST",
+    ),
+}
```

Comment:
These expectation files may get tricky to maintain because some text editors automatically add newlines at the end of a file. Perhaps the `language/tools/move-package/tests/test_runner.rs` should write a newline after the debug format of the package info?